sfdoc/publish/tests/test_integration.py

```diff
@@ -40,6 +41,7 @@
 #
 #   You also probably need to be on VPN to contact your Org.
 
+SHOULD_RUN_INTEGRATION_TESTS = getattr(settings, "RUN_INTEGRATION_TESTS", False)
```

Comment:
Generally defaults for settings should be taken care of in the settings files so that a) the defaults are in one place and b) runtime code can make use of the settings without needing to check if they exist